include/onnxruntime/core/framework/op_kernel_info.h

```diff
@@ -47,6 +47,8 @@ class OpKernelInfo : public OpNodeProtoHelper<ProtoHelperNodeContext> {
 
   common::Status GetFusedFuncs(NodeComputeInfo*& compute_info) const;
 
+  const Path& GetModelPath() const noexcept;
+
  private:
```

Comment:
Is this only needed by ConstantOfShape? If so, can we instead say external data is not supported for that op given it will only be small?

Reply:
Yes right now it is only needed by ConstantOfShape... The reason I added this is today with onnx external data utilities 
 converting a model to external data will convert every single tensor with raw data to external data without any size checks so if we dont support external data for ConstantOfShape then basically we are saying we dont support external data for any model with ConstantOfShape. I am not sure what are all the avenues of generating a model with external data ( I mean do people only use onnx utilities or there are other ways)  and how do they do the conversion... but this seemed like small cost to pay so I added this